README.md

```diff
@@ -1,11 +1,13 @@
 # WireGuard for YunoHost
 
 [![Integration level](https://dash.yunohost.org/integration/wireguard.svg)](https://dash.yunohost.org/appci/app/wireguard) ![](https://ci-apps.yunohost.org/ci/badges/wireguard.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/wireguard.maintain.svg)  
-[![Install WireGuard with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=wireguard)
+[![Install WireGuard with YunoHost](https://install-app.yunohost.org/install-with-yunohost.svg)](https://install-app.yunohost.org/?app=wireguard)
 
 *[Lire ce readme en français.](./README_fr.md)*
 
-:warning: This app is still experimental. WireGuard requires upgrading your Linux kernel to another major version. Check its compatibility before running it on a production system. :warning:
+:warning: This app is still experimental. Check its compatibility before running it on a production system. :warning:
+
+:exclamation: WireGuard for YunoHost will add a DMKS module to your Linux kernel.
```

Comment:
I think we should really consider using the dummy `display_text` arg inside the manifest to display disclaimer like this one (until we have a proper handling of diclaimers) ... we should really avoid assuming people are going to read the README :s

Reply:
Resolved, though the CLI display could be prettier. :p